js/symbol/sprite_atlas.js

```diff
@@ -38,6 +38,8 @@ SpriteAtlas.prototype = {
 };
 
 SpriteAtlas.prototype.resize = function(newRatio) {
+    newRatio = newRatio > 1 ? 2 : 1;
```

Comment:
Is it possible for the ratio to be less than 1 if, say, you zoom out below 100% (with the zoom only text setting disabled)? Or is the ratio independent of the browser's zoom level?